app/scripts/views/connect_another_device.js

```diff
@@ -164,6 +164,10 @@ define(function (require, exports, module) {
       return this.user.isSignedInAccount(this.getAccount());
     },
 
+    _webChannelSupportExists () {
```

Comment:
Can you rename this function to something like `_hasWebChannelSupport` - having a prefix like `is`, `are`, `can` and `has` makes it clear that the function returns a boolean.